helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -204,6 +209,8 @@ spec:
               mountPath: /var/run/secrets/java.io/keystores
         {{- end }}
       volumes:
+      {{- include "data_volume" .Values.repository | nindent 8 }}
+       {{- end }}
```

Comment:
needs to be wrapped with `{{- if .Values.repository.persistence.enabled }}`

check the end that is not indented correctly

Reply:
done like this,

209 {{- if .Values.repository.persistence.enabled }}
210 {{- include "data_volume" .Values.repository | nindent 8 }}
211 {{- end }}